tests/tensor/test_basic.py

```diff
@@ -3984,6 +3985,23 @@ def test_numpy_compare(self):
         assert np.allclose(n_s, t_s)
 
 
+def test_moveaxis():
+    x = at.zeros((3, 4, 5))
+    tuple(moveaxis(x, 0, -1).shape.eval()) == (4, 5, 3)
+    tuple(moveaxis(x, -1, 0).shape.eval()) == (5, 3, 4)
+    tuple(moveaxis(x, [0, 1], [-1, -2]).shape.eval()) == (5, 4, 3)
+    tuple(moveaxis(x, [0, 1, 2], [-1, -2, -3]).shape.eval()) == (5, 4, 3)
+
+
+def test_moveaxis_error():
+    x = at.zeros((3, 4, 5))
+    with pytest.raises(
+        ValueError,
+        match="`source` and `destination` arguments must have the same number of elements",
+    ):
+        moveaxis(x, [0, 1], 0)
```

Comment:
It's not clear what the behavior will be if the indices in `source` and `destination` are not unique, and when they are out of bounds.

Reply:
It will fail. Those checks are done by the numpy helper above `normalize_axis_tuple`. This is basically a copy-paste of the numpy implementation: https://github.com/numpy/numpy/blob/v1.23.0/numpy/core/numeric.py#L1410-L1478